test/packages/parallel/oracle/manifest.yml

```diff
@@ -0,0 +1,40 @@
+format_version: 1.0.0
+name: oracle
```

Comment:
Maybe it's interesting to keep this for reference until `oracle` package is updated in integrations. WDYT ?